modules/ROOT/pages/7.2-release-notes.adoc

```diff
@@ -156,6 +156,15 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 
 // CCFR here.
 
+=== Accessibility for element path buttons, added tooltip to describe the button and removed incorrect `aria-level` attribute.
+// #TINY-10891
+
+Previously in {productname}, an issue was identified where the element path buttons in the status bar did not provide any aria description of their functionality. Without any description for the element path buttons in the status bar, screen reader users were unable to describe the purpose of these buttons to visually impaired users.
+
+To address this issue in {productname} {release-version}, tooltips have been added to the buttons in the element path. These tooltips provide a description of the button's functionality. Additionally, the `aria-describedby` attribute has been added when the tooltip is focused, allowing screen readers to announce and describe the purpose of the button. Now, a tooltip appears when hovering over an element path button with the mouse or focusing on it using the keyboard, and the description is announced by screen readers if the button is focused. This improvement ensures that visually impaired users receive the same information as other users.
+
+Another part of the change is the removal of the incorrect `aria-level` attribute that was added to these buttons. This change was made to ensure that the editor complies with the "Elements must only use supported ARIA attributes" rule in WCAG 2.x (A).
```

Comment:
@shanmen-tiny It is ambiguous to me from the docs notes where this 'rule' is coming from. I added this detail but I'm not sure if it's a good idea to mention WCAG here unless we actually comply to a specific version of WCAG.

Reply:
👍 , perhaps we shouldn't mention that. In that case, maybe we can replace it with something like this?

```
In previous versions of {productname}, Axe-DevTools (web accessibility testing extension) identified that the element path buttons contain an invalid `aria-level` attribute that is not permitted on `role="button"` elements.
```

Reply:
@kemister85 thoughts?

Reply:
@ShiridiGandham

Reply:
@FarzadHayat @shanmen-tiny 

Suggestion:

In previous versions of {productname}, Axe-DevTools (web accessibility testing extension) identified that the element path buttons contain an invalid `aria-level` attribute that is not permitted on `role="button"` elements.

As a consequence, screen readers were unable to describe the purpose of these buttons to visually impaired users.

{productname} {release-version} addresses this issue by adding tooltips to the buttons in the element path. which now provides a description of the button's functionality. In addition, `aria-describedby` attribute has been added when the tooltip is focused, allowing screen readers to announce and describe the purpose of the button when in-focus. Now, a tooltip appears when hovering over an element path button with the mouse or focusing on it using the keyboard.

[NOTE]
With this change {productname} also removed of the incorrect `aria-level` attribute that was added to these buttons previously. This change ensure that the editor complies with the "Elements must only use supported ARIA attributes" rule in WCAG 2.x (A).

Reply:
@shanmen-tiny please review again